packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -37,7 +37,7 @@ const JurisdictionModal = ({
                         toggleModal={toggleJurisdictionModal}
                         type='button'
                         context={context}
-                        width={account_type.type === 'financial' ? '1300px' : '1160px'}
+                        width={account_type.type === 'financial' ? '1280px' : '1100px'}
```

Comment:
@shaheer-deriv 
is this based on figma design?

Reply:
@amina-deriv yes, the output looks similar to the modal in figma design.
this change reduces the white space on the left and right side of the cards.

Reply:
@amina-deriv in figma, the width of modal is 1200px, but giving this exact value in code results in this
![Screenshot 2023-06-06 at 4 57 59 PM](https://github.com/binary-com/deriv-app/assets/122449658/789ccde0-914b-4fb7-9148-6cd6bedde597)

Reply:
@amina-deriv its updated according to figma design